src/wp-includes/block-bindings.php

```diff
@@ -129,3 +129,6 @@ function get_all_registered_block_bindings_sources() {
 function get_block_bindings_source( string $source_name ) {
 	return WP_Block_Bindings_Registry::get_instance()->get_registered( $source_name );
 }
+
+// Add the `uses_context` to the block editor settings to ensure it can be consumed in the client.
+add_filter( 'block_editor_settings_all', array( 'WP_Block_Bindings_Registry', 'add_uses_context_to_editor_settings' ), 10 );
```

Comment:
This should be moved to one of the existing functions that applies the filter. Adding filters is for plugins and themes when they don't have access to internals of WordPress.

Reply:
I moved it to the place where (I believe) the editor settings are initialized: [link](https://github.com/WordPress/wordpress-develop/pull/6456/files#diff-edfc853e670c7bfc5a838dc74f75ba5d8b95cb4adbddf0644d9227bcb692e344R651-R667).